charter.html

```diff
@@ -417,23 +464,24 @@ <h3 id=decisions>Decision Policy</h3>
 <p>Implementation disagreement should not result in
 implementation-defined behavior or optional features.
 
-<p>Specifications should not make references to or rely on specific
```

Comment:
I think "specifications" was fine here. CGs are [explicitly allowed to produce specifications](https://www.w3.org/community/about/agreements/).

Reply:
Personally, I agree. But given the strongly worded feedback in #3, I'd rather avoid the word.

Reply:
Well, community group reports, really.